// kyouka-api/index.js
const { 
	otakudesu,
	covid,
	ongoing,
	komiku,
	tebakgambar,
	surah,
	sholat,
	lirik,
	chara,
	wattpad,
	playstore,
	linkwa,
	pinterest,
	igdl,
	igstory,
	igstalk,
	twitter,
	fbdown,
	youtube,
	ttdownloader,
	hentaivid,
	tikpor,
	wikimedia,
	quotesAnime,
	konachan,
  	xnxxdl,
	xnxxsearch
} = require('./lib/function')

module.exports.xnxxdl = xnxxdl
module.exports.xnxxsearch = xnxxsearch
module.exports.konachan = konachan
module.exports.quotesAnime = quotesAnime
module.exports.wikimedia = wikimedia
module.exports.hentaivid = hentaivid
module.exports.tikpor = tikpor
module.exports.otakudesu = otakudesu
module.exports.covid = covid
module.exports.ongoing = ongoing
module.exports.komiku = komiku
module.exports.tebakgambar = tebakgambar
module.exports.surah = surah
module.exports.sholat = sholat
module.exports.lirik = lirik
module.exports.chara = chara
module.exports.wattpad = wattpad
module.exports.playstore = playstore
module.exports.linkwa = linkwa
module.exports.pinterest = pinterest
module.exports.igdl = igdl
module.exports.igstory = igstory
module.exports.igstalk = igstalk
module.exports.twitter = twitter
module.exports.fbdown = fbdown
module.exports.youtube = youtube
module.exports.ttdownloader = ttdownloader
